kits/arm/ex_mobile_device_control.py

```diff
@@ -111,106 +116,117 @@ def get_ik(xyz_target, rot_target, ik_seed):
   # Start background logging
   if enable_logging:
     group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
-    phone_group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
+    #phone_group.start_log(os.path.join(local_dir, 'logs'), mkdirs=True)
 
   ik_pos = get_ik(xyz_target_init, rot_mat_target_init, ik_seed_pos)
 
   arm_traj = create_trajectory([0, arm_traj_min_duration], np.array([fbk.position, ik_pos]).T)
   fbk_time = fbk.receive_time
   t0 = fbk_time.min()
+  t_start = time()
   t = 0
-
+  #t = 0
+  print("check 1")
   while t < arm_traj.duration:
+
     group.get_next_feedback(reuse_fbk=fbk)
-    t = min((fbk_time - t0).min(), arm_traj.duration)
+    t = time() - t_start
+    #t = min((fbk_time - t0).min(), arm_traj.duration)
 
     pos, vel, accel = arm_traj.get_state(t)
     cmd.position = pos
     cmd.velocity = vel
-
     if enable_effort_comp:
       fbk_position = fbk.position
       dynamics_comp = get_dynamic_comp_efforts(fbk_position, pos, vel, accel, kin)
       grav_comp = get_grav_comp_efforts(kin, fbk_position, gravity_vec)
       cmd.effort = dynamics_comp + grav_comp + effort_offset
 
     group.send_command(cmd)
-
+  
   # Grab initial pose
-  fbk_mobile = mobile_io_controller.current_feedback
+  fbk_mobile = m.fbk
 
   q = fbk_mobile.ar_orientation[0]
   R_init = quat2rot(q)
 
   xyz_init = fbk_mobile.ar_position[0]
   xyz_phone_new = xyz_init
 
-  end_velocities = np.zeros((1, arm_dof_count))
-  end_accels = np.zeros((1, arm_dof_count))
+  end_velocities = np.zeros(arm_dof_count)
+  end_accels = np.zeros(arm_dof_count)
 
-  max_demo_time = inf % sec
-  phone_fbk_timer = tic
+  phone_fbk_timer = perf_counter()
 
   time_last = t0
   arm_trajStartTime = t0
 
-  first_run = true
-
+  first_run = True
+  
+  print("check 2")
```

Comment:
I'm assuming these `check X` print statements were for debugging purposes... We should remove them before merging this in